listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -254,6 +259,7 @@ def user_feed(user_name: str):
 
     users_following = db_user_relationship.get_following_for_user(user['id'])
 
+    # TODO: Remove these listen events from event list after listen events endpoint is active.
```

Comment:
Do you mean when the feed page UI has been updated instead?

Reply:
Yes